libs/expat/Makefile

```diff
@@ -32,7 +31,7 @@ define Package/libexpat
   SECTION:=libs
   CATEGORY:=Libraries
   TITLE:=An XML parsing library
-  URL:=http://expat.sourceforge.net/
+  URL:=https://libexpat.github.io/
```

Comment:
When you go to the old SF webpage, it points you to the link I posted as the new home.

Are you saying they've moved again?  Are they the Flying Dutchmen of OSS projects?